packages/geoprocessing/src/aws/GeoprocessingHandler.ts

```diff
@@ -46,10 +50,21 @@ const WSS_STAGE = process.env.WSS_STAGE || "";
  *
  * @template T the return type of the geoprocessing function, automatically set from func return type
  * @template G the geometry type of features for the geoprocessing function, automatically set from func feature type
+ * @template P extra parameters to pass to geoprocessing function, automatically set from func parameter type
  */
-export class GeoprocessingHandler<T, G = Polygon | LineString | Point> {
+export class GeoprocessingHandler<
+  T = JSONValue,
+  G = Polygon | LineString | Point,
+  P extends Record<string, JSONValue> = Record<string, JSONValue>
```

Comment:
This says extraParams must be a Record (key/value object) where the values can be any JSON object.  The implementer can provide a more specific type (setting explicit keys that will be on the object like `eezs` or `geographies`) but can't be incompatible with this.  In the absence of a type being passed by the implementer it will default to a Record of JSON values.